specify_pose/proposal.md

```diff
@@ -188,19 +161,20 @@ It's unclear which one may be better. In ROS, `rotation` is used for a
 transform, while `orientation` is used for a pose. However, they both appear
 equivalent.
 
-#### 1.1 (A) Values for `//pose/rotation/@type` or (B) (`@rotation_type`)
+#### 1.1  Values for `//pose/@rotation_format` and `//pose/@degrees` 
 
-The values of `@type` (or `@rotation_type`) that are permitted:
+The permutations of `@rotation_format` and `@degrees` that are
+permitted:
 
-* `rpy_degrees` - A 3-tuple representing Roll-Pitch-Yaw in degrees, which maps
-to a rotation as specified here.
+* `@rotation_format="rpy"`, `@degrees="true"` - A 3-tuple representing
+  Roll-Pitch-Yaw in degrees, which maps to a rotation as specified here.
     * This should be used when the rotation should generally be human-readable.
-* `rpy_radians` - Same as `rpy_degrees`, but with radians as the units for each
+* `@rotation_format="rpy"`, `@degrees="false"` - Same as above, but with radians as the units for each
 angle. This is provided for legacy purposes and ease of conversion.
     * It is not suggested to use this for a text-storage format.
     * Same precision as suggested below for quaternions: Use 17 digits of
     precision, and consider separating each value on a new line.
-* `q_wxyz` - Quaternion as a 4-tuple, represented as  `(w, x, y, z)`, where `w`
+* `@rotation_format="q_wxyz"` - Quaternion as a 4-tuple, represented as  `(w, x, y, z)`, where `w`
 is the real component. This should generally be used when the rotation should be
 machine-generated (e.g. calibration artifacts).
```

Comment:
"Should" sounds a bit too opinionated. Some other use cases for quaternions that I can think of are precision and integration with specific tools. I'd recommend rephrasing to be less specific.

Reply:
I changed it to "This is the recommended format for machine-generated files (e.g. calibration artifacts)." How does that sound?